stripe-ui-core/src/main/java/com/stripe/android/uicore/utils/StateFlows.kt

```diff
@@ -13,14 +14,16 @@ import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.flow.combine
 import kotlinx.coroutines.flow.distinctUntilChanged
 import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.flatMapLatest
 
 /**
  * A subclass of [StateFlow] that allows us to turn a [Flow] into a [StateFlow].
  *
  * @param flow The [Flow] that should be converted to a [StateFlow]
  * @param produceValue The producer of the [StateFlow's] value
  */
-private class FlowToStateFlow<T>(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Might be worth adding a comment, and maybe even a deprecated or something to prevent direct usages.